src/components/ProjectPickerModal.jsx

```diff
@@ -37,7 +37,7 @@ export default function ProjectPickerModal({
           >
             {i18next.t('project-picker.tabs.active')}
           </li>
-          {isEmpty(archivedProjects) ? null : (
+          {!isEmpty(archivedProjects) && (
```

Comment:
This looks much better but I'm having a bit of trouble understanding out what's going on here. Why does this render the component rather than just evaluating to a boolean?

Reply:
Oh I see - thanks!